content/content-strategy/manage-content-requests/create-content/content-production-template/cards.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: cards/cards
```

Comment:
Ignore style for now. Will be replaced with attachment partial soon.

Reply:
Trisha: ok may not be suitable for a card as part of several types of tools and templates.